release/rancher/rancher.go

```diff
@@ -209,52 +220,67 @@ func rancherHelmChartVersions(repoURL string) ([]string, error) {
 	return versions, nil
 }
 
-func SetKDMBranchReferences(ctx context.Context, forkPath, rancherBaseBranch, currentKDMBranch, newKDMBranch, forkOwner, githubToken string, createPR, dryRun bool) error {
+func SetKDMBranchReferences(ctx context.Context, forkPath, rancherBaseBranch, newKDMBranch, githubUser, githubToken string, createPR, dryRun bool) error {
 	branchName := "kdm-set-" + newKDMBranch
 	data := SetBranchReferencesArgs{
 		RancherRepoPath:   forkPath,
-		CurrentBranch:     currentKDMBranch,
 		NewBranch:         newKDMBranch,
 		RancherBaseBranch: rancherBaseBranch,
 		DryRun:            dryRun,
 		BranchName:        branchName,
 	}
-	script := cloneCheckoutRancherScript + setKDMBranchReferencesScript + pushChangesScript
 
-	if err := exec.RunTemplatedScript(forkPath, setKDMBranchReferencesScriptFileName, script, data); err != nil {
+	script := cloneCheckoutRancherScript + setKDMBranchReferencesScript + pushChangesScript
+	logrus.Info("running update files and apply updates script...")
+	output, err := exec.RunTemplatedScript(forkPath, setKDMBranchReferencesScriptFileName, script, data)
+	if err != nil {
 		return err
 	}
+	logrus.Info(output)
 
-	if createPR && !dryRun {
+	if createPR {
+		prName := "Update KDM to " + newKDMBranch
+		logrus.Info("creating PR: " + prName)
+		if dryRun {
+			logrus.Info("dry run, PR will not be created")
+			return nil
+		}
 		ghClient := repository.NewGithub(ctx, githubToken)
 
-		if err := createPRFromRancher(ctx, rancherBaseBranch, "Update KDM to "+newKDMBranch, branchName, forkOwner, ghClient); err != nil {
+		if err := createPRFromRancher(ctx, rancherBaseBranch, prName, branchName, githubUser, ghClient); err != nil {
 			return err
 		}
 	}
-
 	return nil
 }
 
-func SetChartBranchReferences(ctx context.Context, forkPath, rancherBaseBranch, currentBranch, newBranch, forkOwner, githubToken string, createPR, dryRun bool) error {
+func SetChartBranchReferences(ctx context.Context, forkPath, rancherBaseBranch, newBranch, forkOwner, githubToken string, createPR, dryRun bool) error {
 	branchName := "charts-set-" + newBranch
 	data := SetBranchReferencesArgs{
 		RancherRepoPath:   forkPath,
-		CurrentBranch:     currentBranch,
 		NewBranch:         newBranch,
 		RancherBaseBranch: rancherBaseBranch,
 		DryRun:            dryRun,
 		BranchName:        branchName,
 	}
 	script := cloneCheckoutRancherScript + setChartBranchReferencesScript + pushChangesScript
-	if err := exec.RunTemplatedScript(forkPath, setChartReferencesScriptFileName, script, data); err != nil {
+	logrus.Info("running update files script")
+	output, err := exec.RunTemplatedScript(forkPath, setChartReferencesScriptFileName, script, data)
+	if err != nil {
 		return err
 	}
+	logrus.Info(output)
 
-	if createPR && !dryRun {
+	if createPR {
+		prName := "Update charts branch references to " + newBranch
+		logrus.Info("creating PR: " + prName)
+		if dryRun {
+			logrus.Info("dry run, PR will not be created")
```

Comment:
We probably want to output something to the user showing what would have been created if it weren't a dry run.